### DISCUSSION TITLE
Tkinter /  Customtkinter

### DISCUSSION THREAD

**Discussion by katelynrohrer** [catppuccin/catppuccin#2452] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://github.com/TomSchimansky/CustomTkinter/blob/master/customtkinter/assets/themes/dark-blue.json

### Any additional comments?

I really want to make my own app with the catppuccin theme! Thank you!

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by scarcekoi** [comment]
Started work on one here: https://github.com/scarcekoi/customtkinter.

**Reply by scarcekoi** [reply]
Done: #3012

**Reply by sgoudham** [reply]
Closing since any further discussion should take place in #3012
